lib/scholar/linear/svm.ex

```diff
@@ -0,0 +1,239 @@
+defmodule Scholar.Linear.SVM do
+  @moduledoc """
+  SVM in both binary and multinomial variants for classification and regression.
```

Comment:
We should add a note for now about our implementation and how it is limited. Any references are also welcome!

Reply:
Is the note that I pushed okay?

Reply:
Yes!